tests/conftest.py

```diff
@@ -130,3 +143,14 @@ def agent_xml():
     obj_xml: The object xml.
     """
     return AgentTestXML()
+
+
+@pytest.fixture
+def premis_schema():
+    """Provides an lxml schema object for the Premis v2 XSD suitable
+    for validation.
+    """
+    schema_file = open(os.path.join(SCHEMA_DIR, "premis-v2-3.xsd"))
```

Comment:
Should this file be opened in a `with` so we don't have to worry about `close()`?